scan-command/src/main/java/io/ballerina/scan/internal/IssueImpl.java

```diff
@@ -45,9 +45,9 @@ public class IssueImpl implements Issue {
               String filePath) {
         LineRange lineRange = location.lineRange();
         TextRange textRange = location.textRange();
-        this.location = new BLangDiagnosticLocation(lineRange.fileName(), lineRange.startLine().line(),
-                lineRange.endLine().line(), lineRange.startLine().offset(), lineRange.endLine().offset(),
-                textRange.startOffset(), textRange.length());
+        this.location = new BLangDiagnosticLocation(lineRange.fileName(), lineRange.startLine().line() + 1,
+                lineRange.endLine().line() + 1, lineRange.startLine().offset() + 1, lineRange.endLine().offset() + 1,
+                textRange.startOffset() + 1, textRange.length());
```

Comment:
We shouldn't change this. We should change the assertion util method instead. Similar to `org.ballerinalang.test.BAssertUtil#validateError(org.ballerinalang.test.CompileResult, int, java.lang.String, int, int)`.